### PR TITLE
Add support for DisAlign(CVPR 2021)

### PR DESCRIPTION
**Pull request by tonysy**
1. Add support for DisAlign(CVPR 2021)
2. Add support for LVIS v0.5/v1.0
3. Add C++ LVIS eval API
4. Fix Bug in Cascade R-CNN
5. Add support for evaluation on dumped predictions without inference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ For all the models supported by cvpods, please refer to [MODEL_ZOO](https://gith
 * [LLA](https://github.com/Megvii-BaseDetection/LLA)
 * [OTA](https://github.com/Megvii-BaseDetection/OTA)
 * [SelfSup](https://github.com/poodarchu/SelfSup)
+* [DisAlign](https://github.com/Megvii-BaseDetection/DisAlign)
```
Comment (poodarchu) [inline README.md:98]: Put this line alphabetically.

File: cvpods/configs/base_config.py [hunk 2]
```diff
@@ -172,6 +172,8 @@
             SCALE_RANGES=(),
         ),
         PRECISE_BN=dict(ENABLED=False, NUM_ITER=200),
+        # If True, evaluate the dumped prediction without run the inference on dataset.
+        ON_FILES=False,
```
Comment (poodarchu) [inline cvpods/configs/base_config.py:176]: put this option to your own configs.

File: cvpods/engine/runner.py [hunk 3]
```diff
@@ -14,6 +14,7 @@
 from cvpods.evaluation import (
     DatasetEvaluator,
     inference_on_dataset,
+    inference_on_files,
```
Comment (poodarchu) [inline cvpods/engine/runner.py:17]: you might put all modifications related to 'inference_on_files' to an individual pr.

File: cvpods/modeling/roi_heads/fast_rcnn.py [hunk 4]
```diff
@@ -397,3 +397,177 @@ def forward(self, x):
         scores = self.cls_score(x)
         proposal_deltas = self.bbox_pred(x)
         return scores, proposal_deltas
+
+
+class DisAlignFastRCNNOutputLayers(nn.Module):
```
Comment (poodarchu) [inline cvpods/modeling/roi_heads/fast_rcnn.py:402]: maybe you can put these modules into another xxx.py?

File: cvpods/layers/wrappers.py [hunk 5]
```diff
@@ -252,6 +252,184 @@ def forward(self, x):
         return x
 
 
+class NormalizedConv2d(torch.nn.Conv2d):
```
Comment (poodarchu) [inline cvpods/layers/wrappers.py:255]: why is this called "Normalized"?
